src/order/complete_lattice.lean

```diff
@@ -58,13 +58,17 @@ theorem Sup_le : (∀b∈s, b ≤ a) → Sup s ≤ a := complete_lattice.Sup_le
 
 theorem le_Inf : (∀b∈s, a ≤ b) → a ≤ Inf s := complete_lattice.le_Inf s a
 
-lemma is_lub_Sup : is_lub s (Sup s) := ⟨assume x, le_Sup, assume x, Sup_le⟩
+lemma is_lub_Sup (s : set α) : is_lub s (Sup s) := ⟨assume x, le_Sup, assume x, Sup_le⟩
 
-lemma is_lub_iff_Sup_eq : is_lub s a ↔ Sup s = a := is_lub_iff_eq_of_is_lub is_lub_Sup
+private lemma is_lub.Sup_eq (h : is_lub s a) : Sup s = a := (is_lub_Sup s).unique h
 
-lemma is_glb_Inf : is_glb s (Inf s) := ⟨assume a, Inf_le, assume a, le_Inf⟩
+alias is_lub.Sup_eq ← is_lub.Sup_eq
 
-lemma is_glb_iff_Inf_eq : is_glb s a ↔ Inf s = a := is_glb_iff_eq_of_is_glb is_glb_Inf
+lemma is_glb_Inf (s : set α) : is_glb s (Inf s) := ⟨assume a, Inf_le, assume a, le_Inf⟩
+
+private lemma is_glb.Inf_eq (h : is_glb s a) : Inf s = a := (is_glb_Inf s).unique h
+
+alias is_glb.Inf_eq ← is_glb.Inf_eq
```

Comment:
What is the point of this `alias` dance? Is it to escape some namespace? I think it would be worth explaining by a comment or a `Note`, here and below.

Reply:
Yes, `lemma` actually defines `lattice.is_glb.Inf_eq`, and it is not useful for dot notation. BTW, why do we need the `lattice` namespace?

Reply:
I'll add a comment later tonight.

Reply:
AFAIK, the lattice namespace is there only for historical reasons. @digama0 comments?

Reply:
The lattice namespace is @johoelzl's doing, but I'm happy to see it gone (especially the `lattice.lattice.foo` thing, but I think that's already fixed).

Reply:
I can do it in another PR.